IDE/MonoDevelop/MonoDevelop.MonoGame/templates/Common/app.manifest

```diff
@@ -0,0 +1,42 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1">
+  <assemblyIdentity version="1.0.0.0" name="${ProjectName}"/>
+  <trustInfo xmlns="urn:schemas-microsoft-com:asm.v2">
+    <security>
+      <requestedPrivileges xmlns="urn:schemas-microsoft-com:asm.v3">
+        <requestedExecutionLevel  level="asInvoker" uiAccess="false" />
+      </requestedPrivileges>
+    </security>
+  </trustInfo>
+
+  <compatibility xmlns="urn:schemas-microsoft-com:compatibility.v1">
+    <application>
+      <!-- A list of the Windows versions that this application has been tested on and is
+           is designed to work with. Uncomment the appropriate elements and Windows will 
+           automatically selected the most compatible environment. -->
+
+      <!-- Windows Vista -->
+      <supportedOS Id="{e2011457-1546-43c5-a5fe-008deee3d3f0}" />
+
+      <!-- Windows 7 -->
+      <supportedOS Id="{35138b9a-5d96-4fbd-8e2d-a2440225f93a}" />
+
+      <!-- Windows 8 -->
+      <supportedOS Id="{4a2f28e3-53b9-4441-ba9c-d69d4a4a6e38}" />
+
+      <!-- Windows 8.1 -->
+      <supportedOS Id="{1f676c76-80e1-4239-95bb-83d0f6d0da78}" />
+
+      <!-- Windows 10 -->
+      <supportedOS Id="{8e0f7a12-bfb3-4fe8-b9a5-48fd50a15a9a}" />
+
+    </application>
+  </compatibility>
+
+  <application xmlns="urn:schemas-microsoft-com:asm.v3">
+    <windowsSettings>
+      <dpiAware xmlns="http://schemas.microsoft.com/SMI/2005/WindowsSettings">true</dpiAware>
```

Comment:
No, it's system dpi aware. Should be "true/pm" for per monitor
https://msdn.microsoft.com/library/aa374191(v=VS.85).aspx
I'll get to this and the other PR's I have open in a couple days, maybe a week. I'm a bit busy atm

On 21 Aug 2016, at 00:31, "Tom Spilman" <notifications@github.com<mailto:notifications@github.com>> wrote:


In IDE/MonoDevelop/MonoDevelop.MonoGame/templates/Common/app.manifest<https://github.com/MonoGame/MonoGame/pull/5096#discussion_r75585797>:

> +
> +      <!-- Windows 8 -->
> +      <supportedOS Id="{4a2f28e3-53b9-4441-ba9c-d69d4a4a6e38}" />
> +
> +      <!-- Windows 8.1 -->
> +      <supportedOS Id="{1f676c76-80e1-4239-95bb-83d0f6d0da78}" />
> +
> +      <!-- Windows 10 -->
> +      <supportedOS Id="{8e0f7a12-bfb3-4fe8-b9a5-48fd50a15a9a}" />
> +
> +    </application>
> +  </compatibility>
> +
> +  <application xmlns="urn:schemas-microsoft-com:asm.v3">
> +    <windowsSettings>
> +      <dpiAware xmlns="http://schemas.microsoft.com/SMI/2005/WindowsSettings">true</dpiAware>


I was reading the following slides:

video.ch9.ms/files/GDC2015-PCGamesforWindows10.pptx

They suggested all DirectX game should be set to "Per Monitor High DPI Aware". Is this the correct setting for that? I mean specifically the "per monitor" part.

-
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/MonoGame/MonoGame/pull/5096/files/f2ef517a116f74b69db9e515a2998b6c3abe8f51#r75585797>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AOL69lZXX1y6qHpq8gXWGK1s7TnOlUpxks5qh4BagaJpZM4Jgkr_>.